src/openrct2-ui/input/Shortcuts.cpp

```diff
@@ -211,7 +212,8 @@ static void ShortcutAdjustWater()
 
     if (!(gScreenFlags & SCREEN_FLAGS_SCENARIO_EDITOR) || gEditorStep == EditorStep::LandscapeEditor)
     {
-        if (!(gScreenFlags & (SCREEN_FLAGS_TRACK_DESIGNER | SCREEN_FLAGS_TRACK_MANAGER)))
+        if (!(gScreenFlags & (SCREEN_FLAGS_TRACK_DESIGNER | SCREEN_FLAGS_TRACK_MANAGER))
+            && (!(gParkFlags & PARK_FLAGS_FORBID_LANDSCAPE_CHANGES) || gCheatsSandboxMode))
```

Comment:
This is the same check as before, and since it's quite big already, please move this to a separate function.